packages/editor/src/components/post-actions/actions.js

```diff
@@ -230,28 +230,30 @@ const trashPostAction = {
 	},
 };
 
-function useCanUserEligibilityCheckPostType( capability, resource, action ) {
+function useCanUserEligibilityCheckPostType( capability, postType, action ) {
 	const registry = useRegistry();
 	return useMemo(
 		() => ( {
 			...action,
 			isEligible( item ) {
 				return (
 					action.isEligible( item ) &&
-					registry
-						.select( coreStore )
-						.canUser( capability, resource, item.id )
+					registry.select( coreStore ).canUser( capability, {
```

Comment:
There's something I don't understand here. how does this value change when it gets resolved. There's nothing triggering a re-render when the value of this selector changes and `isEligible` is not an event callback it's something that is called when rendering.

I'm sure we have a hack elsewhere that "hides" the bug, but this is not great.

Reply:
The original PR (#62589) might have more details, but I agree that this feels super "hacky."